src/webgl/uniforms.js

```diff
@@ -213,42 +224,38 @@ export function getUniformsTable({
 } = {}) {
   assert(program);
 
+  const SHADER_MODULE_UNIFORM_REGEXP = '.*_.*';
+
   const uniformLocations = program._uniformSetters;
-  const table = {[header]: {}};
+  const table = {}; // {[header]: {}};
 
-  // Add program's provided uniforms
-  for (const uniformName in uniformLocations) {
-    const uniform = uniforms[uniformName];
-    if (uniform !== undefined) {
-      table[uniformName] = {
-        Type: uniform,
-        Value: formatValue(uniform)
-      };
+  // Add program's provided uniforms (in alphabetical order)
+  const uniformNames = Object.keys(uniformLocations).sort();
+
+  // First add non-underscored uniforms (assumed not coming from shader modules)
+  for (const uniformName of uniformNames) {
+    if (!uniformName.match(SHADER_MODULE_UNIFORM_REGEXP)) {
```

Comment:
Doesn't have to be this PR, but displaying these uniforms in alphabetically sorted order will be very useful, especially when debugging shaders that have many uniforms, you can easily point to the one you need.
Able to display only a list user provided ones (using something like luma.log.uniformsList = ['uniform1', 'uniform2']) will be even great.

Reply:
>  but displaying these uniforms in alphabetically sorted order will be very useful, especially when debugging shaders that have many uniforms

That is part of this PR, see like 233. It first sorts alphabetically, and then it sorts so that any uniforms that have underscored in them come last (so the shaders own uniforms first, then shader module uniforms).

Unfortunately this didn't fix the corresponding issue in the seer integration which seems to need separate sorting.